### DISCUSSION TITLE
Armor Upgrades usage

### DISCUSSION THREAD

**Discussion by xanstin** [wildj79/foundryvtt-starfinder#489] [Q&A] [answered]
I got the jetpack into the armor slot and added a slot within the jetpack for the battery. Is there a way to actually trigger the usage to track whats left?

**Comment by wildj79** [comment, marked as answer]
Not really. The system will handle things like charges used in a battery for a weapon during combat, but it doesn't handle longer term battery usage. Things like this are generally expected to be handled manually for now.

**Comment by xanstin** [comment]
Is there a way to edit it maybe via macro? It is at 40/40 but in the item i think it lowers the total.
